test/network/exception_test.cpp

```diff
@@ -72,7 +72,7 @@ void *ParserExceptionTest(int port) {
     } catch (const pqxx::pqxx_exception &e) {
       const pqxx::sql_error *s = dynamic_cast<const pqxx::sql_error*>(&e.base());
       if (s) {
-        LOG_INFO("Invalid Drop Query: %s", e.base().what());
+        LOG_TRACE("Invalid Drop Query: %s", e.base().what());
```

Comment:
Maybe switch to `LOG_WARN`?

Reply:
@apavlo We want this test to catch the exception, to check the Peloton reports error when user inputs an invalid query (at parsing stage). Then shall we put `log_warn` here?